AcsEmulator/AcsEmulatorAPI/CallAutomationController.cs

```diff
@@ -9,20 +8,39 @@ public static class CallAutomationController
     {
         public static void AddCallAutomationEndpoints(this WebApplication app)
         {
-            app.MapPost("/calling/callConnections", [Authorize] async (ClaimsPrincipal principal, AcsDbContext db, CreateCallRequest req) =>
+            app.MapPost("/calling/callConnections", async (AcsDbContext db, CreateCallRequest req) =>
```

Comment:
Nits/Nice to Have: Adding a couple lines unit test to just call the endpoint and check the HTTP return code would be a great addition to this PR, you can look at the Identity Unit test

If you use Visual studio you can generate automatically the test file and base structure by right clicking on the method name (AddCallAutomationEndpoints) -> Create Unit Test -> MSTestv2 -> <change New project to EXISTING TEST PROJECT> and that's all

Reply:
Thanks for this suggestion @vincentg ! I added one unit test to test the create code.